src/helpers/types/models.ts

```diff
@@ -69,6 +69,10 @@ export interface CommunityAttributes extends UbiCommunity {
     organization?: UbiOrganization;
 }
 
+export interface CommunityFundraisingUrl {
```

Comment:
Call it `CommunityCampaing` instead. So we match names with models, reducing possible confusion.